assets/src/edit-story/elements/shared/index.js

```diff
@@ -93,3 +93,9 @@ export const elementWithStyle = css`
     letterSpacing ? letterSpacing + 'em' : null};
   text-align: ${({ textAlign }) => textAlign};
 `;
+
+export const MEDIA_MASK_OPACITY = 0.4;
+
+export const SHARED_DEFAULT_ATTRIBUTES = {
```

Comment:
Not sure if there already was logic for shared attributes, LMK if that's the case.